docs/core/usage.qmd

```diff
@@ -625,18 +626,35 @@ node_id               | Int32   | -            | sorted
 
 DiscreteControl is implemented based on [VectorContinuousCallback](https://docs.sciml.ai/DiffEqDocs/stable/features/callback_functions/#VectorContinuousCallback).
 
+## DiscreteControl / variable
+
+The compound variable schema defines linear combinations of variables which can be used in conditions. This means that
+this schema defines new variables with the given `compound_variable_id` that look like
+$$
+\text{weight}_1 * \text{variable}_1 + \text{weight}_2 * \text{variable}_2 + \ldots,
+$$
+
+which can be for instance an average or a difference of variables. If a variable comes from a time-series, a look ahead $\Delta t$ can be supplied.
+
+column               | type     | unit    | restriction
+-------------------- | -------- | ------- | -----------
+node_id              | Int32    | -       | sorted
+compound_variable_id | String   | -       | sorted per node_id
+listen_node_type     | String   | -       | known node type
+listen_node_id       | Int32    | -       | sorted per node_id
+variable             | String   | -       | must be "level" or "flow_rate", sorted per listen_node_id
+weight               | Float64  | -       | (optional, default 1.0)
+look_ahead           | Float64  | $s$     | Only on transient boundary conditions, non-negative (optional, default 0.0).
+
 ## DiscreteControl / condition
 
-The condition schema defines conditions of the form 'the discrete_control node with this node id listens to whether the given variable of the node with the given listen feature id is grater than the given value'. If the condition variable comes from a time-series, a look ahead $\Delta t$ can be supplied.
+The condition schema defines conditions of the form 'the `discrete_control` node with this `node_id`listens to whether the variable given by the `node_id` and `compound_variable_id` is greater than `greater_than`'. If the condition variable comes from a time-series, a look ahead $\Delta t$ can be supplied. Multiple conditions with different `greater_than` values can be defined on the same compound_variable.
 
-column            | type     | unit    | restriction
------------------ | -------- | ------- | -----------
-node_id           | Int32    | -       | sorted
-listen_node_type  | String   | -       | known node type
-listen_node_id    | Int32    | -       | sorted per node_id
-variable          | String   | -       | must be "level" or "flow_rate", sorted per listen_node_id
-greater_than      | Float64  | various | sorted per variable
-look_ahead        | Float64  | $s$     | Only on transient boundary conditions, non-negative (optional, default 0)
+column               | type     | unit    | restriction
+-------------------- | -------- | ------- | -----------
+node_id              | Int32    | -       | sorted
+compound_variable_id | Int32    | -       | -
```

Comment:
Why did we suport multiple compound variables per condition? Is there a specific use case? If so, let's put it in the PR description, so it ends up in the git commit history.

Reply:
Hmm, I believe it follows logically from these tables right? If you can have multiple variables, you have to indicate which are compound, and which are not.

Reply:
We support multiple conditions per `DiscreteControl` node, which do not need to be on the same variable (and now all variables on which a condition is placed is called a compound variable, even if it is effectively a singular variable). Logic like 'do X if the level at basin 1 is at least 10 and the level at basin 2 is at least 10' is quite sensible.